### ISSUE TITLE
Spike on Globus integration for DataSpace

### ISSUE THREAD

**Issue by jrgriffiniii** [pulibrary/dspace-development#86]
https://github.com/globusonline/transfer-api-client-java offers a client which may possibly be used to generate Globus downloads URIs.

**Comment by jrgriffiniii** [comment]
From what I recall, @kmcelwee had advanced this quite a bit along with @kayiwa using https://github.com/pulibrary/princeton_ansible/pull/1717

**Comment by kmcelwee** [comment]
Yes! I looked into this, and I could not find a peer institution who did the same. Francis originally attempted to have Globus share a server with DSpace, but Globus did not play well with DSpace’s required version of Apache. Also, Globus attempted to integrate DSpace, but they abandoned the project in July 2019. [Press release.](https://www.globus.org/blog/globus-publication-new-approach) [Archived github repo.](https://github.com/globus/globus-publish-dspace)

With all that in mind, I ended up recommending to the PRDS team that we not apply any formal integration between DSpace and Globus, at least for now. Instead, I recommended using a Globus link in the metadata.

Here's a link to the full proposal: https://docs.google.com/document/d/1FCr6ADVGenX-Z0SrKhTSZiMKnNmzylyqjry0DSDYdyk/edit#

Let me know if you have any thoughts about this! I'm not sure how this affects #87
